proxy/proxy.go

```diff
@@ -1272,6 +1279,9 @@ func (p *Proxy) errorResponse(ctx *context, err error) {
 	if ctx.response.StatusCode == 499 {
 		msgPrefix = "client canceled"
 		logFunc = p.log.Infof
+		if p.accessLogDisabled {
+			logFunc = p.log.Debugf
+		}
```

Comment:
Why this change in this scope?

Reply:
Because I think we did not caught it the last time and I saw it during tests.